test/JDBC/expected/table_variable_xact_errors_isolation_snapshot.out

```diff
@@ -170,3 +170,171 @@ date#!#date
 DROP TYPE empDates
 GO
 
+-------------------------------------------------------------------------------
+-- BABEL-4225: Error inside function should not cause crash
```

Comment:
we should add following variation:
Table variables with diff attributes like indexes, constraints, custom types (whatever is allowed)
Different kind errors : Ones which result in stmt rollback, batch rollback and may be TRY/CATCH
Different places for errors : inside function, inside procedure, inside batch, sp_execute, exec and cases where procedure call functions

Reply:
I've added some scenarios. Most of these are handled in one of these

```-rw-r--r-- 1 klejao amazon   512 Jun 20 16:26 table-variable-psql.mix
-rw-r--r-- 1 klejao amazon  1212 Jun 20 16:26 table-variable-vu-cleanup.sql
-rw-r--r-- 1 klejao amazon  6565 Jun 20 16:26 table-variable-vu-prepare.sql
-rw-r--r-- 1 klejao amazon  2176 Jun 20 16:26 table-variable-vu-verify.sql
-rw-r--r-- 1 klejao amazon   104 Jun 20 16:26 table_variable_xact_basic_isolation_snapshot.txt
-rw-r--r-- 1 klejao amazon 13444 Jun 20 16:26 table_variable_xact_basic.sql
-rw-r--r-- 1 klejao amazon    81 Jun 20 16:26 table_variable_xact_basic_xact_abort_on.txt
-rw-r--r-- 1 klejao amazon   105 Jun 20 16:26 table_variable_xact_errors_isolation_snapshot.txt
-rw-r--r-- 1 klejao amazon  8297 Jun 21 20:43 table_variable_xact_errors.sql
-rw-r--r-- 1 klejao amazon    82 Jun 20 16:26 table_variable_xact_errors_xact_abort_on.txt
-rw-r--r-- 1 klejao amazon   105 Jun 20 16:26 table_variable_xact_nested_isolation_snapshot.txt
-rw-r--r-- 1 klejao amazon  9034 Jun 20 16:26 table_variable_xact_nested.sql
-rw-r--r-- 1 klejao amazon    82 Jun 20 16:26 table_variable_xact_nested_xact_abort_on.txt```